crates/ironrdp-cliprdr-native/src/windows/cliprdr_backend.rs

```diff
@@ -35,7 +38,9 @@ impl WinCliprdrBackend {
         // Wake up subproc event loop; Dont wait for result
         //
         // SAFETY: it is safe to call PostMessageW from any thread with a valid window handle
-        unsafe { PostMessageW(self.window, WM_CLIPRDR_BACKEND_EVENT, WPARAM(0), LPARAM(0)) };
+        if let Err(err) = unsafe { PostMessageW(self.window, WM_CLIPRDR_BACKEND_EVENT, WPARAM(0), LPARAM(0)) } {
+            tracing::error!("Failed to post message to wake up subproc event loop: {}", err);
```

Comment:
[nit] `use tracing::error`

Reply:
ok

Reply:
note: I’m in favor of adding this in lib.rs

```
#[macro_use]
extern crate tracing;
```

and remove `tracing::` qualification since we use the logging macros pervasively. Does not need to be addressed in this PR.